plugins/kc.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kc
+spec:
+  version: v0.19.3
+  homepage: https://github.com/sunny0826/kubecm
+  shortDescription: "Easier management of kubeconfig."
+  description: |
+    This plugin provide easier management of kubeconfig.
+    We can 'list', 'switch', 'add' and 'delete' kubeconfig ctx more efficiently.
```

Comment:
Don't use words like `easier` or `more efficiently`, the user will be the judge of that. :)
Instead, focus on what the plugin offers to the user. You can look at other plugin descriptions as good examples.

Reply:
Updated. PTAL thanks.